cmd/antrea-agent/options.go

```diff
@@ -146,4 +151,74 @@ func (o *Options) setDefaults() {
 	if o.config.APIPort == 0 {
 		o.config.APIPort = apis.AntreaAgentAPIPort
 	}
+
+	if o.config.FlowCollectorAddr != "" && o.config.PollAndExportInterval == "" {
+		o.pollingInterval = 5 * time.Second
+		o.exportInterval = 60 * time.Second
+	}
+}
+
+func (o *Options) validateFlowExporterConfig() error {
+	if features.DefaultFeatureGate.Enabled(features.FlowExporter) {
+		if o.config.FlowCollectorAddr != "" {
+			// Check if it is TCP or UDP
+			strSlice := strings.Split(o.config.FlowCollectorAddr, ":")
+			var proto string
+			if len(strSlice) == 2 {
+				// No separator "." and proto is given
+				proto = "tcp"
+			} else if len(strSlice) > 2 {
+				if strSlice[2] == "udp" {
+					proto = "udp"
+				} else {
+					// All other cases default proto is tcp
+					proto = "tcp"
+				}
```

Comment:
probably better to return an error if the protocol is neither "udp" nor "tcp"

Reply:
Done.